src/app/layouts/dashboard/mentee/recommendations.js

```diff
@@ -70,8 +70,8 @@ const Recommendation = ({ setShowDetailView, setCurrentMentor }) => {
 
               <VStack display="inline">
                 {map(({ label, attribute }) => (
-                  <HStack justifyContent="space-between" key={`${label}-${attribute}`}>
-                    <Text>{label}</Text>
+                  <HStack justifyContent="" key={`${label}-${attribute}`}>
+                    <Text minW={'114px'}>{label}</Text>
```

Comment:
use chakra UI's units where possible. 1 unit in Chakra UI = 4px. (Not sure though, check documentation).